themes/qgis-theme/docs_index.html

```diff
@@ -89,7 +89,8 @@ <h3>{{ _('QGIS 3.10') }}</h3>
             <div class="offset1 span5">
                 <h3>{{ _('QGIS testing') }}</h3>
                 <p>{{ _("We are still updating (not translating yet) the documentation for releases newer than QGIS 3.10. We call this version 'QGIS testing' and the documentation can be found here: ") }} <a href="https://docs.qgis.org/testing" target="_blank" class="reference external">{{ _('https://docs.qgis.org/testing') }}</a></p>
-                <p><a href="https://docs.qgis.org/testing/en/docs/user_manual" target="_blank" class="reference external">{{ _('User guide/Manual') }}</a></p>
+                <p><a href="https://docs.qgis.org/testing/en/docs/user_manual" target="_blank" class="reference external">{{ _('Desktop User guide/Manual') }}</a></p>
+		<p><a href="https://docs.qgis.org/testing/en/docs/server_manual" target="_blank" class="reference external">{{ _('Server guide/Manual') }}</a></p>
                 <p><a href="https://docs.qgis.org/testing/en/docs/training_manual/" >{{ _('QGIS Training manual') }}</a></p>
```

Comment:
Let's capitalize every word? We are not consistent currently
```suggestion
                <p><a href="https://docs.qgis.org/testing/en/docs/user_manual" target="_blank" class="reference external">{{ _('Desktop User Guide/Manual') }}</a></p>
		<p><a href="https://docs.qgis.org/testing/en/docs/server_manual" target="_blank" class="reference external">{{ _('Server Guide/Manual') }}</a></p>
                <p><a href="https://docs.qgis.org/testing/en/docs/training_manual/" >{{ _('QGIS Training Manual') }}</a></p>
```